### ISSUE TITLE
Undefined codesystem

### ISSUE THREAD

**Issue by jkiddo** [HL7/vulcan-eproduct-info#71]
https://github.com/HL7/vulcan-eproduct-info/blob/90d1c29227241a09add4fb8013bdede5a19ce94a/input/fsh/examples/karvea-ema/ClinicalUseDefinition-interaction.fsh#L7 uses an undefined codesystem

**Comment by cander2** [comment]
Yes, most are undefined. Will have to work with G-SRS and FHIR HTA to get them registered.

Going through them now and notice that a few are actually registered so i'll make those changes to make life easier in the meantime. For instance,
Use this instead of the G-SRS one http://fdasis.nlm.nih.gov since it is already registered (https://terminology.hl7.org/CodeSystem-UNII.html)

**Comment by jkiddo** [comment]
seems sane - there are also quite a few more undefined ones - you can see those on http://build.fhir.org/ig/HL7/vulcan-eproduct-info/qa.html - look for `Unable to provide support for code system`
